### PR TITLE
[changelog] updated changelog

### PR DESCRIPTION
**Pull request by roboquat**
Updated the changelog from recent PR descriptions

```release-note
NONE
```
/werft no-preview
/werft no-test

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+- | `/{project}/.gitpod/jetbrains/options`<br>Example: `/workspace/gitpod/.gitpod/jetbrains/options` | Each JetBrains product configured in a Git repo of your project. |  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- | `/{project}/.gitpod/jetbrains/{product}/options`<br>Example: `/workspace/gitpod/.gitpod/jetbrains/intellij/options` | A specific product configured in a Git repo of the project. | ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+-  ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- For an example see https://github.com/akosyakov/parcel-demo/tree/jb-proxy-test applying proxy settings via Docker image. ([#14566](https://github.com/gitpod-io/gitpod/pull/14566)) - [@akosyakov](https://github.com/akosyakov)
+- The processes in the workspace are given up to 3 minutes after receiving SIGTERM. ([#14498](https://github.com/gitpod-io/gitpod/pull/14498)) - [@utam0k](https://github.com/utam0k)
```
Comment (felladrin) [inline CHANGELOG.md:23]: @akosyakov, it seems the Release Notes don't accept multiple lines as in https://github.com/gitpod-io/gitpod/pull/14566.

**Review by felladrin — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (akosyakov) [reply CHANGELOG.md:23]: Cc @filiptronicek @loujaybee 

we can adjust PR, Lou will need to put it better next week anyway)

Comment (felladrin) [reply CHANGELOG.md:23]: Let's adjust it today, otherwise, @roboquat will create a new PR with this same content tomorrow.

**Comment by felladrin** [comment]
Closing it in favor of a new PR with the correct updates [[1](https://github.com/gitpod-io/gitpod/pull/14725#discussion_r1024090735)] that will be created tomorrow.

**Pull request closed without merging**